modelmatrix-core/src/main/scala/com/collective/modelmatrix/db/SchemaInstaller.scala

```diff
@@ -0,0 +1,27 @@
+package com.collective.modelmatrix.db
+
+import org.flywaydb.core.Flyway
+
+/**
+ * Created by jpocalan on 7/27/15.
+ */
+trait SchemaInstaller {
+  val dbConfigWrapper: DatabaseConfigWrapper
+
+  def installOrMigrate: Boolean = {
```

Comment:
It's scala good practice to add parenthesis to method that changes some external state: http://docs.scala-lang.org/style/method-invocation.html (Arity-0)